app/src/i18n/locales/en/data.json

```diff
@@ -161,6 +161,10 @@
   "delete-activity": "Delete activity",
   "delete-activities-prompt": "Are you sure you want to <2>permanently delete </2> all activities in the commercial and institutional buildings sub-sector from the city's inventory data?",
   "delete-activity-prompt": "Are you sure you want to <2> permanently delete </2> this activity from the city's inventory data?",
+  "delete-activity-success": "Activity deleted successfully",
+  "all-activities-deleted": "All activities deleted successfully",
+  "delete-activity-failed": "Failed to delete activity",
+  "delete-all-activities-failed": "Failed to delete all activities",
```

Comment:
Can you ask the product team to provide the Spanish translations for these? Thanks :sparkles:

Reply:
Doesn't have to be provided in this PR, just so they are aware. Might make sense to start maintaining a shared list of missing translation strings, although there's probably tooling for i18next to extract them.

Reply:
Reached out to Greta about this